src/Microsoft.ML.TensorFlow/TensorFlow/TensorflowUtils.cs

```diff
@@ -25,7 +30,92 @@ public static void Initialize()
             ImageAnalytics.Initialize();
         }
 
+        private static unsafe ISchema GetModelSchema(IExceptionContext ectx, TFGraph graph)
```

Comment:
>unsafe [](start = 23, length = 6)

just curious, why it's unsafe?

Reply:
Thanks for pointing it out, it doesn't need to be. In a previous iteration I was directly running the while loop over the graph that is now in the Graph class, so it was unsafe.

---
In reply to: [218204434](https://github.com/dotnet/machinelearning/pull/862#discussion_r218204434) [](ancestors = 218204434)